client/src/Components/Pages/Questionnaire/PersonalDetails1/index.js

```diff
@@ -1,65 +1,158 @@
-    { value: 'technology', label: 'technology' },
+    { value: 'History', label: 'History' },
+    { value: 'Geography', label: 'Geography' },
+    { value: 'Design & technology', label: 'Design & technology' },
+    { value: 'Business Studies', label: 'Business Studies' },
+    { value: 'Chemistry', label: 'Chemistry' },
+    { value: 'Classics', label: 'Classics' },
+    { value: 'Computer Science', label: 'Computer Science' },
+    { value: 'Drama', label: 'Drama' },
+    { value: 'Economics', label: 'Economics' },
+    { value: 'Business Management', label: 'Business Management' },
+    { value: 'Health and Social Care', label: 'Health and Social Care' },
+    { value: 'ICT', label: 'ICT' },
+    { value: 'Latin', label: 'Latin' },
+    { value: 'Law', label: 'Law' },
+    { value: 'Media Studies', label: 'Media Studies' },
+    { value: 'Modern Foreign Languages', label: 'Modern Foreign Languages' },
+    { value: 'Music', label: 'Music' },
+    { value: 'Physical Education', label: 'Physical Education' },
+    { value: 'Politics', label: 'Politics' },
+    { value: 'Psychology', label: 'Psychology' },
+    { value: 'Religious Studies', label: 'Religious Studies' },
+    { value: 'Philosophy', label: 'Philosophy' },
+    { value: 'Social Sciences', label: 'Social Sciences' },
+    { value: 'Statistics', label: 'Statistics' },
+    { value: 'Welsh', label: 'Welsh' },
+    { value: 'Other', label: 'Other' },
+
   ];
 
+  const fileMaxSize = 100000000  //byte
+
 class PersonalDetails1 extends Component {
 
     state = {
-        selectedOption: null, 
+        subjects: null, 
+        favSubjects: null,
+        enjoySchool: '',
+        hobbies: '',
+        futureJob: '',
+        interestedJob: '',
+        files:[],
+        label:"choose files"
     }
+    
+    handleSubjectChange = (subjects)=> {
+        this.setState({subjects}); 
+      }
 
-    handleChange = (selectedOption) => {
+    handleFavChange = (favSubjects)=> {
+        this.setState({favSubjects});
+      }
 
-        this.setState({ selectedOption });
+    onChange = (e) => {
+        const { name, value } = e.target;
+        this.setState({ 
+            [name]: value
+        });
+      };
+    
+    handleSubmit= (event) => {
+        event.preventDefault();
+      
+ 
       }
+    
+    handleOnDrop = (files, rejectedFiles) => {
+     console.log(files);
+     
+     
+
+        if (rejectedFiles && rejectedFiles.length > 0) {
+            const currentRejectFile = rejectedFiles[0];
+            const currentRejectFileSize = currentRejectFile.size;
+            
+            
+            if ( currentRejectFileSize > fileMaxSize) {
+                alert('This file is too big..please choose another one')
+            }
+        }
+            const currentFile = files[0];
+            const currentFileSize = currentFile.size;
+            if ( currentFileSize > fileMaxSize) {
+                alert('This file is too big..please choose another one')
```

Comment:
Same thing here as I mentioned above, it might be helpful to tell the user what the max size is.